lib/pr_checker.js

```diff
@@ -230,6 +239,33 @@ class PRChecker {
     // 3. is not authored by the people opening the PR
     return true;
   }
+
+  checkCommitsAfterReview(commits, reviews, logger) {
+    const commitIndex = commits.length - 1;
+    const reviewIndex = reviews.length - 1;
+    const lastCommit = commits[commitIndex].commit;
+    const lastReview = reviews[reviewIndex];
+
+    const commitDate = lastCommit.committedDate;
+    const reviewDate = lastReview.publishedAt;
+    if (commitDate > reviewDate) {
+      warn();
```

Comment:
I would prefer if both warn and logCommitTitles were unwound (that is, their contents were just within this if statement). Since they're not called more than once, they're just adding to the call stack unnecessarily and each is creating an extra closure.